src/components/Timeline.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <div class="timeline">
```

Comment:
Right now this component seems too simple to be its own component, but it is supposed to be more complex later (it'll have its own logic, style and controls)